From 8b9070610cf6c0e0da6ffe0dd0ba3a23f14e144c Mon Sep 17 00:00:00 2001
From: auto commit <unknown@unknown>
Date: Fri, 24 Oct 2008 12:22:50 -0700
Subject: [PATCH] cavium: CFLAGS for the GPL executive Makefile.

Adjust the EXTRA_CFLAGS for building with "make O=somedir ...".
Also hide the output of some of the other mundane commands.

Signed-off-by: Paul Gortmaker <Paul.Gortmaker@windriver.com>
---
 arch/mips/cavium-octeon/gpl-executive/Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/mips/cavium-octeon/gpl-executive/Makefile b/arch/mips/cavium-octeon/gpl-executive/Makefile
index 4b6e788..8e6cdf6 100644
--- a/arch/mips/cavium-octeon/gpl-executive/Makefile
+++ b/arch/mips/cavium-octeon/gpl-executive/Makefile
@@ -11,7 +11,7 @@
 
 source:=$(srctree)/$(src)
 
-EXTRA_CFLAGS += -I $(obj)/config -I $(source)/config
+EXTRA_CFLAGS += -I $(obj)/config -I $(source)/config -I $(OCTEON_ROOT)/executive
 
 executive-files := cvmx-bootmem.o
 executive-files += cvmx-cmd-queue.o
-- 
1.5.5.1

